src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
```

Comment:
1. Exception Handling: The method `throwError()` intentionally throws a `RuntimeException`. While this can be useful for testing and review purposes, it is not recommended to leave intentional exceptions in production code. It's better to handle exceptions gracefully, for example, by returning a meaningful error message to the client.